com.photopea.Photopea.yml

```diff
@@ -0,0 +1,22 @@
+app-id: com.photopea.Photopea
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: photopea
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --filesystem=xdg-pictures
+modules:
+  - name: photopea
+    buildsystem: simple
+    build-commands:
+    - install -Dm755 -t /app/bin photopea
+    - install -Dm755 -t /app/share/applications photopea.desktop
```

Comment:
you'll need a `rename-desktop-file`

Reply:
also there are no icons installed. you need a 128x128 icon or scalable SVG to be installed. (and properly named)